### ISSUE TITLE
Entry-point link from README.md not found

### ISSUE THREAD

**Issue by MikeRalphson** [darosh/openapi-directory-lite#1]
Should it point to https://raw.githubusercontent.com/darosh/openapi-directory-lite/master/index.json ?

Interesting project! Do you want to set up any kind of web-hook to keep this repository in-sync with changes in openapi-directory? An alternative might be a Travis cron job?

**Comment by darosh** [comment]
@MikeRalphson  good catch, it should be https://darosh.github.io/openapi-directory-lite/index.json

The "in-sync" sounds interesting, but right now I have no exact idea what is the scope/purpose/future of this repo. 

I use it in the https://github.com/darosh/oax project. There is integrated directory browser in left panel with category filter and I am about to add static full text search from this (openapi-directory-lite) repo https://darosh.github.io/openapi-directory-lite/search/ 

I am still exploring options and I guess I will see in a couple of week how much I like the fulltext or how much I do not like having  separate spec repo etc.

![image](https://user-images.githubusercontent.com/297320/30922264-e79ebaca-a3a8-11e7-9fcf-7c65883a3594.png)

![image](https://user-images.githubusercontent.com/297320/30922342-20583d0a-a3a9-11e7-807a-e2cfdb954b97.png)
